contracts/FYToken.sol

```diff
@@ -232,8 +252,9 @@ contract FYToken is IFYToken, IERC3156FlashLender, AccessControl(), ERC20Permit,
     {
         require(token == address(this), "Unsupported currency");
         _mint(address(receiver), amount);
+        uint128 fee = _flashFee(amount).u128();
         require(receiver.onFlashLoan(msg.sender, token, amount, 0, data) == FLASH_LOAN_RETURN, "Non-compliant borrower");
```

Comment:
s/0/fee/

Reply:
@alcueca

Reply:
Oh shit.

Oh well, that means that we can't enable the flash loans on the june series unless we do it without a fee.